conf/messages

```diff
@@ -218,18 +218,23 @@ taxCal_breakdown.additional_charges                                 = Additional
 taxCal_breakdown.table.capital_gains_tax                            = Capital Gains Tax
 taxCal_breakdown.additional_deductions                              = Additional deductions
 taxCal_breakdown.table.ZRT                                          = Zero rate ({0} at {1}%)
+taxCal_breakdown.table.nic4.ZRT                                     = Class 4 National Insurance ({0} at {1}%)
 taxCal_breakdown.table.SSR                                          = Starting rate ({0} at {1}%)
 taxCal_breakdown.table.SRT                                          = Starter rate ({0} at {1}%)
 taxCal_breakdown.table.BRT                                          = Basic rate ({0} at {1}%)
+taxCal_breakdown.table.nic4.BRT                                     = Class 4 National Insurance ({0} at {1}%)
 taxCal_breakdown.table.IRT                                          = Intermediate rate ({0} at {1}%)
 taxCal_breakdown.table.HRT                                          = Higher rate ({0} at {1}%)
+taxCal_breakdown.table.nic4.HRT                                     = Class 4 National Insurance ({0} at {1}%)
 taxCal_breakdown.table.ART                                          = Additional rate ({0} at {1}%)
 taxCal_breakdown.table.ART_scottish                                 = Top rate ({0} at {1}%)
 taxCal_breakdown.table.ZRTBR                                        = Basic rate band at nil rate ({0} at {1}%)
 taxCal_breakdown.table.ZRTHR                                        = Higher rate band at nil rate ({0} at {1}%)
 taxCal_breakdown.table.ZRTAR                                        = Additional rate band at nil rate ({0} at {1}%)
 taxCal_breakdown.table.totalIncomeTaxAndNicsDue                     = Income Tax and National Insurance contributions due
 taxCal_breakdown.table.nic4                                         = Class 4 National Insurance ({0} at {1}%)
+taxCal_breakdown.table.BRT.uplift                                   = 1.25% uplift in National Insurance contributions funds NHS, health and social care
+taxCal_breakdown.national_insurance_contributions                   = National Insurance contributions
```

Comment:
Maybe worth checking with design team about character **I** being capital in National Insurance but not in National insurance type.

Reply:
Done, thanks